lib/registration.pm

```diff
@@ -96,11 +96,11 @@ sub fill_in_registration_data {
                         assert_and_click "scc-code-field-$addon";
                     }
                     type_string $regcode;
+                    send_key 'alt-n';
```

Comment:
Surely you at least want to have the alt-n AFTER it's taken the screenshot on line 101?

That screenshot is there so we can be sure it's typed the right Regcode..now we'll never see it..